docs/upgrading/from_seat_3_0.md

```diff
@@ -0,0 +1,187 @@
+# SeAT 3.0 to 4.0
+
+The upgrade path from SeAT 3.0 to SeAT 4.0 requires some manual work. This is mainly due to a large package revamp that
+have been made in SeAT 4.0.
+
+!!! info
+
+    Major changes have been applied in the way packages are working together, especially regarding eveapi, web and services ones.
+    Also, the way jobs are queued and queues themselve have been altered to reduce resources thumbrint and improve fluency.
+    
+    Before starting upgrade, pay attention on plugins you may use and ensure they are compatible with the new generation.
+    If it should not be the case, please come to us on Slack - so we will attempt to assist or redirect you to proper person.
+
+!!! warning
+
+    Before starting to do anything, be sure you read and understand this whole page content.
+
+    Also, remember to do at least a complete backup of your current database that you pick out of server on which SeAT
+    is currently running, together with the `.env` file. Both of them are the only things required to rebuilt your instance
+    in case of failure.
+    
+    In all provided instructions, we assume you were using a standalone installation using official deployment instructions.
+    Pay attention if you made changes on your own and adapt provided instructions accordingly.
+    
+    Last but not least, we assume you are authenticated against your server as root user.
+    
+## Requirements
+
+- Check [SeAT 4.0 requirements]
+- A database backup is an absolute **must**. Everything in SeAT can be recovered in some way or form except *for your database*.
```

Comment:
Should we mention backing up the `.env` file here as well to reinforce the point?

Reply:
I don't think it's a "critical" need - but that's indead a thing we can add to avoid rand complaints :p